specs/_features/ePBS/validator.md

```diff
@@ -0,0 +1,243 @@
+    validator = state.validators[validator_index]
+    committee = get_beacon_committee(state, slot, index)
+    min_balance_increments = validator.effective_balance // MIN_ACTIVATION_BALANCE
+    committee_balance_increments = get_total_balance(state, set(committee)) // MIN_ACTIVATION_BALANCE
+    denominator = committee_balance_increments ** min_balance_increments
+    numerator = denominator - (committee_balance_increments -  TARGET_AGGREGATORS_PER_COMMITTEE) ** min_balance_increments
+    modulo = denominator // numerator
+    return bytes_to_uint64(hash(slot_signature)[0:8]) % modulo == 0
+```
+
+Rest of the aggregation process remains unchanged.
+
+### Payload timeliness attestation
+
+Some validators are selected to submit payload timeliness attestation. The `committee`, assigned `index`, and assigned `slot` for which the validator performs this role during an epoch are defined by `get_execution_committee(state, slot)`.
+
+A validator should create and broadcast the `execution_attestation` to the global execution attestation subnet at `SECONDS_PER_SLOT * 3 / INTERVALS_PER_SLOT` seconds after the start of `slot`
+
+#### Constructing payload attestation
+
+##### Prepare payload attestation message
+If a validator is in the payload attestation committee (i.e. `is_assigned_to_payload_committee()` above returns True), 
+then the validator should prepare a `PayloadAttestationMessage` for the current slot,
+according to the logic in `get_payload_attestation_message` at `SECONDS_PER_SLOT * 3 / INTERVALS_PER_SLOT` interval.
+
+```python
+def is_assigned_to_payload_committee(state: BeaconState,
+                                  slot: Slot,
+                                  validator_index: ValidatorIndex) -> bool:
+                                                     
+    committe = get_ptc(state, slot)
+    return validator_index in committee
+```
+
+Next, the validator creates `payload_attestation_message` as follows:
+* Set `payload_attestation_data.slot = slot` where `slot` is the assigned slot.
+* Set `payload_attestation_data.beacon_block_root = block_root` where `block_root` is the block hash seen from the block builder reveal at `SECONDS_PER_SLOT * 2 / INTERVALS_PER_SLOT`. If the `SignedExecutionPayloadEnvelope`
+* Set `payload_attestation_message.validator_index = validator_index` where `validator_index` is the validator chosen to submit. The private key mapping to `state.validators[validator_index].pubkey` is used to sign the payload timeliness attestation.
+* Set `payload_attestation_message = PayloadAttestationMessage(data=payload_attestation_data, signature=payload_attestation_signature)`, where `payload_attestation_signature` is obtained from:
+
+```python
+def get_payload_attestation_message_signature(state: BeaconState, attestation: PayloadAttestationMessage, privkey: int) -> BLSSignature:
+    domain = get_domain(state, DOMAIN_PAYLOAD_TIMELINESS_COMMITTEE, compute_epoch_at_slot(attestation.slot))
```

Comment:
in https://github.com/potuz/consensus-specs/blob/epbs/specs/_features/epbs/beacon-chain.md#domain-types there's `DOMAIN_PTC_ATTESTER`